src/pybind/mgr/dashboard_v2/controllers/host.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from ..tools import ApiController, RESTController
+
+
+@ApiController('host')
+class Host(RESTController):
```

Comment:
```
@ApiController('host')
 class Host(RESTController):
     def list(self):
         return self.mgr.list_servers()

     def get(self, key):
         return self.mgr.get_server(key()

```

Reply:
why add the `get` method implementation if the `get` is not used by the frontend! Please don't add code that is not used.

Reply:
I'd say, because in a typical RESTful API, you can access individual elements by calling `/collection/<key>`

Reply:
we are not implementing a REST API, we are implementing the dashboard and all the required services that the backend needs to provide to the frontend.

Reply:
But we are implementing a restful API. At least partially. And we should IMO aim for not breaking that design philosophy unnecessary. And implementing `get` is really super super easy here.